src/commonMain/resources/db/migration/V63__BotMapRedesign.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE public.aiDeclarationType AS ENUM
+(
+    'Admin',
+    'Uploader',
+    'SageScore',
+    'None'
+);
+
+ALTER TABLE beatmap
+    ADD "declaredAi" aiDeclarationType;
```

Comment:
Should this have
```sql
NOT NULL DEFAULT 'None';
```

The code doesn't mark this as nullable but the default it null and nothing shows up for me locally.